lib/nexthop.h

```diff
@@ -134,7 +139,7 @@ struct nexthop {
 	/* Encapsulation information. */
 	enum nh_encap_type nh_encap_type;
 	union {
```

Comment:
Is this info exclusive with anything else in the nexthop struct - the label info, for example? if so, we should consider creating a union so that we don't increase the overall size.